quattor/functions/network.pan

```diff
@@ -214,3 +214,41 @@ function get_subnet_params = {
 
     error(format("No subnet matching address %s found", ipaddr));
 };
+
+@documentation{
+    descr = Convert a network prefix (0..32) into a netmask (IPv4 notation), e.g. 16 becomes 255.255.0.0
+    arg = prefix
+}
+function subnet_prefix_to_mask = {
+    pref = ARGV[0];
+    if (!(is_long(pref) && pref >= 0 && pref <= 32)) {
```

Comment:
There is a function for this `is_ipv4_prefix_length`

Reply:
@stdweird any objection to using the existing function?

Reply:
done